src/explore-education-statistics-frontend/src/modules/methodologies/components/MethodologyList.tsx

```diff
@@ -1,29 +1,38 @@
-import Link from '@frontend/components/Link';
 import React from 'react';
 import { Publication } from '@common/services/publicationService';
+import Link from '@frontend/components/Link';
 
 interface Props {
   publications: Publication[];
 }
 
 function MethodologyList({ publications }: Props) {
+  const methodologySlugs: string[] = [];
+  const methodologies = publications.map(({ methodology }) => {
+    if (methodology && !methodologySlugs.includes(methodology.slug)) {
```

Comment:
This still seems to be attempting to remove some duplicates but I didn't think this would be required if the new hierarchy shows methodologies by their parent Publication?